.env.example

```
@@ -7,4 +7,11 @@ NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=notakto-3359b.firebasestorage.app
 FIREBASE_PRIVATE_KEY="-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCchTQXJj3Ss+eJ\nZxy3iel5JICDGc4z30pY6hisUkAdBN6DyYOpjLType9x1/EH1H2qLjBUHB62YLAK\nPZEIB8dSr1W1Ysn0BmekNF1xoPMAW+2mXqdFvfu08j1f9NBuN6mt3VMty5NRnrzg\nmON+wg19TAW0Gn7J9kXaWWKgO7erL8DT/xfsnhsOtSzWRjcHhw6jWDzrUfgmk3+/\nQ8a3MJghjMD5bZXs29minUHKnIe3/kkAmA9P0j3x39il/BRP2B9443MsjpJI3PlC\ntyQEZnf5S0HhcFB2937GXMd73oJgTDVDQ/NAGy01ZA/CSNbsIDXpg10zKMJ4uzCe\nOQ/TlGvPAgMBAAECggEACYL+xg6zixkWZP84e4mMqqlRNfuyJZeC3ncQxh1peq9P\nu7/Z7x/Mj0JjlA4vaO5nhJlco9U2+JZU8bUJb0IpwW3uTiDmwkbxXW/hOIUzaejW\n6a6u3kohVb8ILnX42fnVUeo1TSnQlhEzdo1T+zayYymPidZiW3D1OxkGKVP3tsC5\nKYbV+7uSpnD3oFspcD+ZA8y/aZM8f/lIRLrUWlMl2NDetX8ngnnXP/0iHwaxMK2Z\nLI9x+IVMEnHWRgR7HYJqpkXvqGbcZUSQw5vD3QwwNqtxxkgq5A7DIlw1DrzgJGah\nH38HV/nLi/uxnNqBp/AVA06vbvouEAWdpFqXdFtQQQKBgQDIWVhhRTyq0BX5PGgb\nTAhQFjbjj1W07mI1Ybg95LkDxPOchhl+nwATIRCogaNCsnMnqo1MwOvvR+Jxvs9H\nn/VFSjTL1JSWXUzVWQl5ZwZYxb3hzMoO0ounqGNpc6/6gIzUXf+LLOW0UNz6cmqs\nqonRbbS9QKI80guk1cI1QiX3pwKBgQDH/zyOjOIvaHhqc+SBrFguhTfExPmO3KGM\nUH1+uHHpeAU/H7t40Sgy4//SWqmM4ffbLTTveDwlMxSU75GD95u1ByJ88OIbXZMC\nZB3NW87bZYpC6nGO7va74GUMxBf6CU7AEAYsfREUE0kaE4YoCsvPp11RgaJlHPqa\nI/tdl65vmQKBgQCJyWiO/R8MGrACCRlQLKpK6fw15I39MTU5oUskxfi4Luub6l3+\nRzu17DeYrA+r/y5TSWiEMOoSVk3WDHRhvhDg5ngI9tp5zxDYabOqWsByrQw6Dq2o\nJAUH+Tblfzq3/VDdfQgSn34a1MIlPCtBNMgz0E9NGPZ6/bokUx6NokCaYQKBgBzd\nnuzoMqErD+41ae65ZETan3D8CRTwjTSGTYajsnVDCGbS+qj2bznhJIRE56HY34A3\n5yMFFJE97SON7E/Cxit7qyOjVM8kU1qN/qssLBAIeMstsVQy81OskIkmKFvDSiV4\napuzNWrTWcK0vLE81avT+SkRb9HuO9FRji80/E1JAoGAITTSi6PGIdqH96VIDxYR\nQJVoexaXv6GEVzPb6vsKivNqPakXSpyyJDN9Z+RNxBDTI5MPqhc1qEYTWxXvIaVv\nbootstYYYebPrPdoNt5IGy0tUowSYTanxwW+haSnnZfJhnKn0oH88g9AlyDYOfLI\nAPUn5NyMo1QBGq4L2Y3IJp4=\n-----END PRIVATE KEY-----\n"
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The FIREBASE_PRIVATE_KEY key should go before the NEXT_PUBLIC_FIREBASE_PROJECT_ID key

(UnorderedKey)